run-e2e.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# run the script to run e2e tests for cluster loader, netperf and dns. 
+# This enables the same to be invoked from kubetest from test-infra.
+set -o nounset
+set -o pipefail
```

Comment:
nit: empty line after those

Reply:
Removed empty line

Reply:
Please add  empty line after that line.

Same with above comment.

Reply:
Added